docs/pages/desktop-access/reference/configuration.mdx

```diff
@@ -55,14 +55,26 @@ kind: windows_desktop
 metadata:
   name: fixed-screen-size
 spec:
-  host_id: 307e091b-7f6b-42e0-b78d-3362ad10b55d
   addr: 192.168.1.153:3389
   non_ad: true
+  # host_id is the `metadata.name` of the `windows_desktop_service` resource
+  # reporting this desktop.
+  #
+  # For AD-joined hosts, this value MUST be the `name` of the `windows_desktop_service`
+  # resource that connects to the AD domain via LDAP. For hosts that use local accounts
+  # (`non_ad: true`), this value can be the `name` of any `windows_desktop_service`
```

Comment:
This part is incorrect. `host_id` behaves exactly the same for AD and non-AD, there is no difference.

It _must_ point to a w_d_s that has connectivity to the desktop.

Reply:
I think there is a difference, for AD we have to target wds with LDAP configured for this domain. For non-AD we can target any wds that has connectivity to the desktop and I think the comment was to point that out (it might need rephrasing though)

Reply:
Any wds with connectivity will work here too.

Reply:
cf01b7998f8c554920e8a275565e0c6f4e71764d